### ISSUE TITLE
Foregrounded mode

### ISSUE THREAD

**Issue by valenvb** [owenthereal/upterm#74]
I'd like to suggest adding a foreground mode, similar to TMate's `-F` option, to allow clients to connect without requiring further interaction on the host side. This would make upterm useful in automated systems/CI environments where it's not always possible to provide input.

**Comment by owenthereal** [comment]
Yes, I would like a foreground mode to help with automation like https://github.com/lhotari/action-upterm/issues/1.
